results.py

```diff
@@ -38,6 +39,8 @@ def to_dict(self) -> dict:
         'trial': self.trial,
         'fuzz_target_source': self.fuzz_target_source,
         'build_script_source': self.build_script_source,
+        'author': str(self.author),
+        'agent_dialogs': self.agent_dialogs,
```

Comment:
how about just chat_history ?

Reply:
Yes! I already named that step, will find a way to extract it later.